wear/src/main/java/io/homeassistant/companion/android/home/views/SensorManagerUi.kt

```diff
@@ -28,7 +32,9 @@ fun SensorManagerUi(
             },
             timeText = { TimeText(!scalingLazyListState.isScrollInProgress) }
         ) {
-            val availableSensors = sensorManager.getAvailableSensors(LocalContext.current)
+            val availableSensors = sensorManager
+                .getAvailableSensors(LocalContext.current)
+                .sortedBy { stringResource(it.name) }
```

Comment:
This doesn't change so I believe it can be stored in a `by remember { ... }`

Also shouldn't the sensor _managers_ be sorted by name as well?

Reply:
> This doesn't change so I believe it can be stored in a `by remember { ... }`

looks like we can save it, will update



> Also shouldn't the sensor _managers_ be sorted by name as well?

its already being done [here](https://github.com/home-assistant/android/blob/7c792c4d4bada99cc35cf6378b000d82b4002df8/wear/src/main/java/io/homeassistant/companion/android/home/views/SensorsView.kt#L69)